anchor-reference-server/src/main/java/org/stellar/anchor/reference/config/AppSettings.java

```diff
@@ -11,4 +11,7 @@ public class AppSettings {
   String version;
   String platformApiEndpoint;
   String hostUrl;
+  String distributionWallet;
+  String distributionWalletMemo;
+  String distributionWalletMemoType;
```

Comment:
Do you want to add some default values here?

Reply:
The default values are `null` and we are expecting `null` as 
```java
    if (!Objects.toString(appSettings.getDistributionWalletMemo()).isEmpty()
        && !Objects.toString(appSettings.getDistributionWalletMemoType()).isEmpty()) {
```